Terminal.Gui/Views/TextView.cs

```diff
@@ -4466,12 +4465,12 @@ private bool DeleteTextForwards ()
         }
         else
         {
-            _historyText.Add ([[.. currentLine]], CursorPosition);
+            _historyText.Add ([ [.. currentLine]], CursorPosition);
```

Comment:
I doubt this is the right format for this expression and all the others similar.

Reply:
Sorry, autoformatter, fixed